demo/pages/index.module.css

```diff
@@ -1,3 +1,3 @@
-.newComponent {
-    border: 1px solid red;
+.container {
+    background-color: #CDCDCD;
```

Comment:
Nitpick alert: I'd drop this, doesn't add much 😛

Reply:
This is for the demo, it's not related to the component itself. I think it might be helpful while making changes and playing around with the props to see how the image behaves.
One good case scenario for this is while playing with `cropRadius` prop. The default color of `backgroundColor` for `png` and `webp` formats is transparent as you can see [here](https://www.contentful.com/developers/docs/references/images-api/#/reference/image-manipulation/background-color). 

Having this `background-color: #CDCDCD` on the container might help to understand the difference between having a solid background color or having a transparent background color.

Does it make sense? 😬

Reply:
Sure, I get it! Although I think that if someone is going to play around in the demo they would eventually change that too! 

I don't have a strong opinion here, carry on 👍